frontend/src/containers/Collection/CollectionDetail/CollectionDetail.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-await-in-loop */
```

Comment:
I think it's related with performance issue. Maybe we should refactor it later. Did you check it is working appropriately with many replies? You can refer to [here](https://eslint.org/docs/rules/no-await-in-loop) for it.

Reply:
Maybe I think it's also inefficient in that frontend call APIs a lot in order to get what frontend actually has already.

Reply:
It worked on 5 pages of replies. I wanted to use the suggested way from eslint, but I could not apply the way, because we need to update state for every api call. And I chose to call apis because if I just updated frontend state about add/delete, duplicate errors occurred with view more action.

Reply:
However it would be much better if we could find another solution.ㅠ

Reply:
OK, it can be tackled later for refactoring.